phidgets_high_speed_encoder/msg/encoder_params.msg

```diff
@@ -0,0 +1,5 @@
+Header header
+int32 index
+int32 count
+int32 count_change
+int32 time
```

Comment:
Also, I think it would be better to use a `sensor_msgs/JointState` message instead and remove this message altogether.